cerberus-web/src/test/java/com/nike/cerberus/service/AuthenticationServiceTest.java

```diff
@@ -428,7 +416,8 @@ public void test_that_findIamRoleAssociatedWithSdb_returns_empty_optional_when_r
       e = e2;
     }
 
-    assertTrue(e instanceof ApiException);
+    IsInstanceOf isInstanceOfException = new IsInstanceOf(ApiException.class);
+    assertTrue(isInstanceOfException.matches(e));
```

Comment:
This approach seems more verbose than just `instanceof`

Reply:
I'll see if I can find the error again but this is a mother case where the findbugs scan was failing. We might want to have a discussion around how much we want to adhere to the advice given by Findbugs.